test/unit/math/gpu/opencl_context_test.cpp

```diff
@@ -0,0 +1,78 @@
+#ifdef STAN_OPENCL
+#include <stan/math/gpu/opencl_context.hpp>
+#include <gtest/gtest.h>
+#include <vector>
+#include <fstream>
+
+TEST(MathGpu, getInfo) {
+  cl::Context cl = stan::math::opencl_context.context();
+  EXPECT_NE("", stan::math::opencl_context.description());
+  EXPECT_NE("", stan::math::opencl_context.capabilities());
+  // auto foo = stan::math::opencl_context.capabilities();
+  // std::cout << foo << std::endl;
+  EXPECT_EQ(256, stan::math::opencl_context.max_workgroup_size());
+  cl::Context cv = stan::math::opencl_context.context();
+  cl::CommandQueue cq = stan::math::opencl_context.queue();
+  std::vector<cl::Device> dv = stan::math::opencl_context.device();
+  std::vector<cl::Platform> pl = stan::math::opencl_context.platform();
```

Comment:
If these are meant to test that the creation is ok, then I'd either split out the test so these are in a separate, well-named test or add something like `SUCCESS() << "can access context, queue, device, and platform";` so it's clear that this is part of the test.

Reply:
I split the creation in a separate test.